src/modules/user/user.service.ts

```diff
@@ -176,4 +176,46 @@ export default class UserService {
 
     return { is_active: user.is_active, message: 'Account Deactivated Successfully' };
   }
+
+  // This UserService class allows only the super admin to get all users
```

Comment:
Is it the class or the method... Rename the method to something more descriptive like getUsersByAdmin.